File: matiasmasso/Mat.NET/DT2/DTODatagridviewCell.vb
Public Class DTODatagridviewCell

    Property FirstDisplayedScrollingRowIndex As Integer
    Property RowIndex As Integer
    Property ColumnIndex As Integer
    Property SortedColumn As Integer
    Property SortOrder As SortOrders

    Public Enum SortOrders
        None
        Ascending
        Descending
    End Enum
End Class

File: matiasmasso/Mat.NET/DTO/DTODatagridviewCell.cs
namespace DTO
{
    public class DTODatagridviewCell
    {
        public int FirstDisplayedScrollingRowIndex { get; set; }
        public int RowIndex { get; set; }
        public int ColumnIndex { get; set; }
        public int SortedColumn { get; set; }
        public SortOrders SortOrder { get; set; }

        public enum SortOrders
        {
            None,
            Ascending,
            Descending
        }
    }
}
